src/pages/Home.jsx

```diff
@@ -1,11 +1,16 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import cart from './../img/cart.png'
 
 export default class Home extends Component {
   render() {
     return (
-      <label htmlFor="search-input" data-testid="home-initial-message">Digite algum termo de pesquisa ou escolha uma categoria.
-        <input id="search-input" />
-      </label>
+      <div>
+        <label htmlFor="search-input" data-testid="home-initial-message">Digite algum termo de pesquisa ou escolha uma categoria.
+          <input id="search-input" />
```

Comment:
oi, @rafMewtwo! acredito que esteja faltando o `type` deste `input` aqui, concorda?

```suggestion
          <input type="text" id="search-input" />
```